ColabTool/LinkPost/index.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { PropTypes } from 'react';
+import {
+  Card,
+  Image,
+  LinkifiedText,
+  Text
+} from '@bufferapp/components';
+import style from './style.css';
+import Post from '../Shared/Post';
+
+// TODO: Padding between inner/outer card 0.5 for left, right, bottom?
+const LinkPost = ({
+  isConfirmingDelete,
+  isDeleting,
+  isWorking,
+  hovered,
+  imageSrc,
+  links,
+  linkAttachment,
+  manager,
+  onApproveClick,
+  onCancelConfirmClick,
+  onDeleteClick,
+  onDeleteConfirmClick,
+  onEditClick,
+  onMouseEnter,
+  onMouseLeave,
+  profile,
+  text,
+}) => {
+  const children = (
+    <div className={style['post-content']}>
+      <div className={style['post-content-text']}>
+        <LinkifiedText
+          links={links}
+        >
+          {text}
+        </LinkifiedText>
+      </div>
+      <div className={style['link-attachment-container']}>
+        <Card
+          hovered={hovered}
+          onMouseEnter={onMouseEnter}
+          onMouseLeave={onMouseLeave}
+          noPadding
+        >
+          <div className={style['link-attachment-content']}>
+            <div className={style['link-attachment-image']}>
+              <Image
+                src={imageSrc}
+                width={'15em'}
+                height={'100%'}
+                maxWidth={'15em'}
+                maxHeight={'15em'}
+                border={'rounded'}
+              />
+            </div>
+            <div className={style['link-attachment-text']}>
+              <span className={style['link-title']}>
```

Comment:
This layout looks a bit odd to me. Using spans and order seems like it would make it more difficult to maintain vs using `div`s to create the hierarchy of the layout.

```html
<div> --- display: flex & flex-direction: row
  <div> --- this is the width of image
    <Image />
  </div>
  <div> --- display: flex & flex-grow: 1
    <div>
      <Text />
    </div>
    <div>
     <Text />
    </div>
  </div>
</div>
```

Would love to hear your thought's on using order too, could be missing some context!